flexgen/apps/data_wrangle/README.md

```diff
@@ -0,0 +1,18 @@
+# FlexGen for Data Wrangling Tasks.
+
+Here we show how to use FlexGen for the data wrangling tasks. The implementation follows the [fm_data_tasks](https://github.com/HazyResearch/fm_data_tasks) repo from [HazyResearch](https://github.com/HazyResearch).
+
+## Install
+
+- Run our install.sh scripts to install additional python library and download the desired datasets.
+
+
+## Examples
+
+- To check the outcome and verify the result of a data imputation task (e.g., Restaurant), run:
+
+      bash test_single_query_case.sh
+
+- To test FlexGen Throughput of a data imputation task (e.g., Restaurant), run:
+
+      bash test_batch_query.case.sh
```

Comment:
Why use different naming style "test_single_query_case.sh" and "test_batch_query.case.sh"?